packages/bootstrap-4/src/ErrorList/ErrorList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import Card from "react-bootstrap/Card";
+import ListGroup from "react-bootstrap/ListGroup";
+
+import { ErrorListProps } from "@rjsf/core";
+
+const ErrorList = ({ errors }: ErrorListProps) => (
+  <Card border="danger" className="mb-2">
```

Comment:
Need more margin between this and the form title

```suggestion
  <Card border="danger" className="mb-4">
```